### ISSUE TITLE
GT.save cuts off output

### ISSUE THREAD

**Issue by lwanger** [posit-dev/great-tables#480]
## Description

Bottom row of my table is cut-off in the output using GT.save. Same result with both pdf and png output files.

## Reproducible example

from pathlib import Path
import pandas as pd

data = {'A': [1,2,3], 'B': ['foo', 'bar', 'baz'], }
df = pd.DataFrame(data)
tbl = (gt.GT(df))

out_file = str(Path("table_export.pdf"))
tbl.save(file=out_file, web_driver='edge')

## Expected result

Bottom row should be shown in full.

## Development environment

- Operating System: Windows
- **great_tables** Version: 0.10
 
[table_export.pdf](https://github.com/user-attachments/files/17234041/table_export.pdf)

![Image](https://github.com/user-attachments/assets/3fd1f69e-8fca-4459-b1bb-910689919487)

**Comment by HansMellman** [comment]
I've had this happen countless times, only anecdotal solution was to ensure that I wasn't throttling my internet connection with anything else at the same time - it was almost like the image was unable to render entirely before the screenshot was taken. I'm speaking as a layperson though because I have no idea if that's actually likely to affect the process or not - just my experience.

**Comment by yk2684** [comment]
Yeah, this also happens to me time to time. My current solution is to use the window_size parameter and keep increasing the size until this no longer occurs. It also helps to run the code multiple times.

**Comment by HansMellman** [comment]
> Yeah, this also happens to me time to time. My current solution is to use the window_size parameter and keep increasing the size until this no longer occurs. It also helps to run the code multiple times.

Rich advised me in the Discord to test using Firefox instead of Chrome and I have not actually had this issue since switching over - which browser were you using in your .save?

**Comment by lwanger** [comment]
I generally use Microsoft Edge on Windows (which is built on the same engine as Chrome)

**Comment by HansMellman** [comment]
> I generally use Microsoft Edge on Windows (which is built on the same engine as Chrome)

Definitely worth giving it a go with Firefox/GeckoDriver, if this issue is persistent for you.
